extension/env/development.env.js

```diff
@@ -1,6 +1,10 @@
 var merge = require("webpack-merge");
 var prodEnv = require("./production.env");
-var common = require("./common").dev;
+try {
+    var common = require("./common").dev;
```

Comment:
mmmh, this looks weird, why are we wrapping a require in try catch?

Reply:
Ah, I actually wanted a workaround so the project would still compile with default settings without adding a custom common.js ENV file to the project, but I agree I don't like this code... should I just revert or do you have suggestions ?

Reply:
I don't know enough about this particular config to suggest an improvement, I would have to look into it. If it works as is let's merge